ojd_daps_skills/pipeline/skill_ner/prodigy/process_data.py

```diff
@@ -0,0 +1,106 @@
+"""
+Process a dataset of job adverts for labelling in Prodigy
```

Comment:
I remember you saying that you cleaned the data slightly more than for the company descriptions approach - do you think its worth me using this instead to be more consistent?

Reply:
Sorry this is probably an out of date reply since you've already started labelling the company descriptions. Might have been worth adding more cleaning, however it might not make much of a difference to your task especially if you are tagging the whole job advert at more or less the sentence level. So I wouldn't worry about it.

The reason I did more cleaning here was because I was splitting into chunks of 5 sentences by full stop, and I kept finding that the previous level of cleaning hadn't separated them out well enough (e.g. it was replacing \n with a space not a full stop).